city_scrapers/spiders/chi_ssa_27.py

```diff
@@ -0,0 +1,170 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+""" from pjsier: #we'll want to parse the agenda links from them too.
+ We also want to get the separate committee meetings and minutes as well.
+ You can get committee meetings and descriptions with div[pageareaid="2"]
+ li:last-child p and minutes with div[pageareaid="Sidebar"] li:last-child a
+"""
+
+
+class ChiSsa27Spider(CityScrapersSpider):
+    name = "chi_ssa_27"
+    agency = "Chicago Special Service Area #27 Lakeview West"
+    timezone = "America/Chicago"
+    allowed_domains = ["lakeviewchamber.com"]
+    start_urls = ["https://www.lakeviewchamber.com/ssa27"]
+
+    # main_page_title = response.css("title::text").get()
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping needs.
+        """
+        items, minutes_pdf, meeting_location, meeting = [], "", "", None
+        the_url_source = response.url
+
+        items_list = (response.css(".panel-body").getall()[4]).split("<p>")
+
+        for i in items_list:
+            items.append(re.sub("\n|\r|</p>", "", i))
+
+        for item in items:
+            if re.search(r'<div', item):
```

Comment:
We should try to avoid parsing HTML directly with regular expressions. You can use `' ' .join(item.css('*::text').extract())` to pull out all nested text in an element

Reply:
Will do.